package.json

```diff
@@ -6,7 +6,8 @@
     "test": "vitest --run test && yarn lint",
     "build": "vite build",
     "prepack": "yarn build",
-    "prepublish": "yarn test"
+    "prepublish": "yarn test",
+    "postinstall": "husky install"
```

Comment:
i don't think this is compatible for public packages, see: https://typicode.github.io/husky/getting-started.html#yarn-2

Reply:
Is is not. I've now added pinst to account for that.

Reply:
it is so hacky :(